### ISSUE TITLE
sqlite constraint error

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#3363]
I just noticed the following errors in my logs:

```
[19188:19215:0214/132659.796643:ERROR:database.cc(1838)] publisher_info_db sqlite error 1299, errno 0: NOT NULL constraint failed: activity_info.score, sql: UPDATE activity_info SET duration=?, score=?, percent=?, weight=? WHERE publisher_id=? AND category=? AND month=? AND year=? AND reconcile_stamp=?
[19188:19215:0214/132911.277113:ERROR:database.cc(1838)] publisher_info_db sqlite error 1299, errno 0: NOT NULL constraint failed: activity_info.score, sql: UPDATE activity_info SET duration=?, score=?, percent=?, weight=? WHERE publisher_id=? AND category=? AND month=? AND year=? AND reconcile_stamp=?
```

## Brave version (brave://version info)

```
Brave	0.59.35 Chromium: 72.0.3626.81 (Build officiel) (64 bits)
Révision	ac8b982e05014492d1bd7d317628a4f22a97ffa0-refs/branch-heads/3626@{#796}
Système d'exploitation	Linux
```

No idea whether this kind of bug report is useful to the team, but I figured I should file it just in case. Happy to provide the affected sqlite files if needed.

**Comment by NejcZdovc** [comment]
@fmarier can you print screen your auto-contribute settings?

**Comment by fmarier** [comment]
![auto-contribute-settings](https://user-images.githubusercontent.com/167821/52891273-d69f2c80-313e-11e9-89d7-e248a90db3e4.png)

**Comment by NejcZdovc** [comment]
@fmarier this is a known issue and was fixed in 0.60. Closing as dupe of https://github.com/brave/brave-browser/issues/3320
